event-app-ui/src/components/Profile/Profile.jsx

```diff
@@ -21,6 +20,8 @@ export default function Profile() {
   const [onGoingEvents, setOnGoingEvents] = useState([]);
   const [futureEvents, setFutureEvents] = useState([]);
   const [pastEvents, setPastEvents] = useState([]);
+  const [isEventWeatherDetailOpen, setIsEventWeatherDetailOpen] =
+    useState(false);
```

Comment:
This can be moved to EventSection component because you're not using the state in this component and only passing it to that one

Reply:
I need to call the state in the profile component to so it can be called independently